### PR TITLE
Updated README links

### PR DESCRIPTION
**Pull request by jennuine**
Updated README links to point to gazebo-classic pages.

### CODE REVIEW

**Comment by quarkytale** [comment]
Install link is still broken, this [install from source link](https://classic.gazebosim.org/tutorials?tut=install_from_source&cat=install#InstallGazebofromsource(UbuntuandMac)) works, can we add it somehow?

File: README.md [hunk 1]
```diff
@@ -4,13 +4,13 @@ Gazebo - A dynamic multi-robot simulator
...
 
-  http://gazebosim.org
+  http://classic.gazebosim.org
 
 Installation
 ------------
 Instructions are located at
 
-  http://gazebosim.org/install
+  http://classic.gazebosim.org/install
```
Comment (iche033) [inline README.md:13]: I think this link is broken? Could be an issue with the website

Comment (jennuine) [reply README.md:13]: Yes, unfortunately it has been broken for awhile. There is a slack thread regarding this. How about I leave the link there for when it gets fixed and add the install from source link @quarkytale suggested?

Comment (quarkytale) [reply README.md:13]: I'm curious how the uninstall link goes to 'uninstall from source' but not the install one

**Review by quarkytale — COMMENTED** [review]
LGTM apart from the install link, [slack thread for reference ](https://osrf.slack.com/archives/C8LMU4733/p1654281183352419)

I'm curious as why the uninstall link directs to 'uninstall from source' but not the install one

File: README.md (hunk 1 above)
Comment (iche033) [reply README.md:13]: here's the redirect: https://github.com/osrf/gazebo_web_app/blob/master/config/routes.rb#L48

Comment (iche033) [reply README.md:13]: we probably just need to update the redirect link there

**Review by iche033 — APPROVED** [review]
looks good to me. The broken gazebo install link is a separate issue.

File: README.md (hunk 1 above)
Comment (quarkytale) [reply README.md:13]: I can take that up

**Comment by jennuine** [comment]
Here's the issue for cross-referencing: https://github.com/osrf/gazebo_web_app/issues/38

**Pull request merged**